app/views/hosts/_list.html.erb

```diff
@@ -31,8 +33,16 @@
         </td>
         <td class="hidden-xs ellipsis"><%= (icon(host.operatingsystem, :size => "16x16") + " #{host.operatingsystem.to_label}").html_safe if host.operatingsystem %></td>
         <td class="hidden-tablet hidden-xs ellipsis"><%= host.compute_resource_or_model %></td>
+        <td class="hidden-tablet hidden-xs ellipsis">
+          <% if host.owner_type == 'User' %>
+            <%= icon_text('user', host.owner, :kind => 'fa') %>
+          <% else %>
+            <%= icon_text('users', host.owner, :kind => 'fa') %>
+          <% end %>
+        </td>
         <td class="hidden-tablet hidden-xs"><%= label_with_link host.hostgroup, 23, @hostgroup_authorizer %></td>
         <td class="hidden-tablet hidden-xs ellipsis"><%= last_report_column(host) %></td>
+        <td class="hidden-tablet hidden-xs" title="<%= host.comment.truncate(255) %>"><%= icon_text('comment', '') unless host.comment.empty? %></td>
```

Comment:
Since we use icon only and we cannot shrink the column a bit more, maybe it's better to actually put text there? Truncated, of course, and leave the full text to be shown on hover.

Or at least we could center the icon like with `Power` column.